src/material/slider/slider.scss

```diff
@@ -115,6 +119,17 @@ $_mat-slots: (tokens-mat-slider.$prefix, tokens-mat-slider.get-token-slots());
   }
 }
 
+.mdc-slider__value-indicator {
```

Comment:
This should be moved into the existing `.mdc-slider__value-indicator`.

Reply:
Hi Cris I pushed the change. Moved the styling to the existing ### **.mdc-slider__value-indicator**